docs/install.rst

```diff
@@ -252,7 +256,13 @@ Building the documentation is typically not necessary unless you are
 developing code or documentation or do not have internet access, because
 the stable, latest, and archived versions of Astropy's documentation are
 available at `docs.astropy.org <https://docs.astropy.org>`_ . The process
-is described in `Building the Documentation from Source <https://docs.astropy.org/en/latest/development/docguide.html#builddocs>`_.
+is described in :ref:`builddocs`.
+
+{%else%}
+
+See the `latest documentation on how to build astropy from source <https://docs.astropy.org/en/latest/install.html#build-from-source>`_.
+
+{%endif%}
 
```

Comment:
Difficult to review if the logic works. Can you open/render it against the bugfix branch?

Reply:
I don't think bug fix branch gonna cut it. But I can open a dummy PR with hacked `conf.py` tonight when I get the chance.

Reply:
OK here. This PR

https://astropy--18556.org.readthedocs.build/en/18556/install.html#build-from-source

Simulated "stable" view

https://astropy--18557.org.readthedocs.build/en/18557/install.html#build-from-source

Reply:
Thanks for entertaining my request. It all looks good!